lm_eval/tasks/selfcheckgpt/selfcheckgpt.yaml

```diff
@@ -0,0 +1,28 @@
+task: selfcheckgpt
+dataset_path: potsawee/wiki_bio_gpt3_hallucination
+output_type: generate_until
+training_split: null
+validation_split: evaluation
+doc_to_text: !function utils.doc_to_text
+doc_to_target: !function utils.doc_to_target
+process_results: !function utils.process_results
+generation_kwargs:
+  # until:
+  #   - "\n"
+  temperature: 0.0
+  do_sample: false
+generation_kwargs_sampling_number: 5
+generation_kwargs_sampling:
```

Comment:
what is the distinction between `generation_kwargs_sampling` and `generation_kwargs`? I am a bit confused by the code creating requests that uses this new arg.

Reply:
> what is the distinction between `generation_kwargs_sampling` and `generation_kwargs`? I am a bit confused by the code creating requests that uses this new arg.

Hi @haileyschoelkopf, Thank you for your reply. The key distinction is that: 

1. For some tasks, we need different generation_kwargs. So I added a new generation_kwargs_sampling here. You can find that generation_kwargs_sampling has different temperature parameters compared with generation_kwargs.